sphinx/cheat-sheet.rst

```diff
@@ -31,36 +31,36 @@ Starting Suites
          cylc run <name>
      - ::
 
-         # run the suite in the current directory
+         # run the suite in the current directory:
          rose suite-run
 
-         # run using a custom name
-         rose suite-run --name <name>
-
-         # run a suite in another directory
+         # run a suite in another directory:
          rose suite-run --path <path>
 
+         # run using a custom name:
+         rose suite-run --name <name>
+
 .. _Stopping Suites:
 
 Stopping Suites
 ^^^^^^^^^^^^^^^
 
 ::
 
-   # Wait for running / submitted tasks to finish then shutdown the suite:
+   # Wait for running/submitted tasks to finish then shut down the suite:
```

Comment:
A noun yes, but not a verb (see e.g. https://www.merriam-webster.com/dictionary/shutdown). I am risking my 'cool' reputation to point this out, so it would not be worth it if I wasn't confident!

Reply:
Wow, minus many cool points :laughing:, OK.